implementations/rust/ockam/ockam/Cargo.toml

```diff
@@ -18,13 +18,22 @@ version = "0.24.0-dev"
 default = ["std", "ockam_transport_tcp", "software_vault", "noise_xx"]
 software_vault = ["ockam_vault", "ockam_vault_sync_core", "ockam_vault_sync_core/software_vault", "ockam_channel/software_vault"]
 noise_xx = ["ockam_key_exchange_xx", "ockam_channel/noise_xx"]
-std = ["ockam_node", "serde/std"]
-alloc = ["ockam_core/alloc", "serde/alloc"]
-no_std = ["ockam_core/no_std", "serde"]
+
+# Option (enabled by default): "std" enables functionality expected to
+# be available on a standard platform.
+std = ["ockam_core/std", "ockam_node/std", "serde/std", "ockam_node_attribute/std"]
+
+# Option: "no_std" enables functionality required for platforms
+# without the standard library.
+no_std = ["ockam_core/no_std", "ockam_node/no_std", "serde"]
+
+# Option: "alloc" enables support for heap allocation on "no_std"
+# platforms, requires nightly.
+alloc = ["ockam_core/alloc", "ockam_node/alloc", "serde/alloc", "ockam_node_attribute/no_std"]
 
 [dependencies]
-ockam_core = { path = "../ockam_core", version = "0.25.0-dev"         }
-ockam_node = { path = "../ockam_node", version = "0.23.0-dev"        , optional = true }
+ockam_core = { path = "../ockam_core", version = "0.25.0-dev", default-features = false }
+ockam_node = { path = "../ockam_node", version = "0.23.0-dev", default-features = false, optional = true }
```

Comment:
I know it's not introduced by this PR, but why is it optional?

Reply:
Probably because the original assumption was that there would be a `ockam_node` and a `ockam_node_no_std`. 

We can probably drop this.

Reply:
I think ockam crate makes no sense without any node as a dependency, so unless it's temporary convenient for your work, l vote drop.